Backpack-SwiftUI/DynamicStack/Classes/BPKDynamicStack.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+import Backpack_Common
+
+struct BPKDynamicStack<Content: View>: View {
+    @Environment(\.dynamicTypeSize) private var dynamicTypeSize
+
+    private let horizontalAlignment: HorizontalAlignment
+    private let verticalAlignment: VerticalAlignment
+    private let spacing: BPKSpacing
+    private let threshold: DynamicTypeSize
+    @ViewBuilder var content: () -> Content
+
+    public init(
+        horizontalAlignment: HorizontalAlignment = .center,
+        verticalAlignment: VerticalAlignment = .center,
+        spacing: BPKSpacing = .none,
+        threshold: DynamicTypeSize = .accessibility1,
```

Comment:
could this be a modifier instead of an initializer prop? seems like an accessory thing to set rather than a hard requirement.

Reply:
Also giving it a more explicit name, threshold of what ?